var/spack/repos/builtin/packages/py-cf-units/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyCfUnits(PythonPackage):
+    """Units of measure as required by the Climate and Forecast (CF)
+    metadata conventions.
+    """
+
+    homepage = "https://scitools.org.uk"
+    url      = "https://pypi.io/packages/source/c/cf-units/cf-units-2.1.1.tar.gz"
+    git      = "https://github.com/SciTools/cf-units.git"
+
+    version('master', branch='master')
+    version('2.1.1', extension='tar.gz', sha256='fa0ef8efd84546e61088aa23e76ebbaf7043167dc3a7f35f34549c234b543530')
```

Comment:
just fyi: the extensions keyword should not be needed

Reply:
I'll merge once the `extension` keyword is removed.